values/ccd-data-store-api.yaml.gotmpl

```diff
@@ -1,7 +1,7 @@
 app:
   name: ccd-data-store-api
   group: backend
-  replicaCount: 2
+  replicaCount: 1
```

Comment:
Would be good to know when this started failing from since it looks like we've been running 2 replicas for a while and it ought to work like that.